src/components/Calendar/arrow-left.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24">
```

Comment:
I'm still not sure how we should be handling optimization of these SVGs. We could optimize them before saving them, or rely on optimization through the loader (it has [support for svgo](https://github.com/boopathi/react-svg-loader/tree/master/packages/react-svg-loader#svgo-options)).

I haven't done anything regarding this with any of the icons included in existing components.

Reply:
Hm, I passed all svg's trough svgo, but for some reason it left the xmlns attr (with `remove xmlns` checked). 

btw, In my previous company I've spent some some time building svg pipeline for automatic minification/customisation of raw svg files and directly exporting them as themable components without requiring any build step/loader from potential user of the library. We may discuss this in person in the following days :)

Reply:
That sounds pretty cool! I put it on my TODO.